Proxy request.signal through in vite dev (#9976)
remix-run/remix: .changeset/hungry-experts-suffer.md

---
"@remix-run/dev": patch
---

> Abort request.signal in vite dev when the node response is closed

Author: ```suggestion
Properly abort `request.signal` during `vite dev` when the node response is closed
```
